### PR TITLE
Install binder binds initial children

### PR DESCRIPTION
**Pull request by saantiaguilera**
This PR fixes behavior shown in #10 :)

### CODE REVIEW

**Comment by saantiaguilera** [comment]
@rjrjr who should I ask to see the PR ? (sorry to bother you, you are the last commiter haha)

File: coordinators/src/main/java/com/squareup/coordinators/Coordinators.java [hunk 1]
```diff
@@ -48,6 +48,9 @@ public static void bind(View view, CoordinatorProvider provider) {
    * to the group.
    */
   public static void installBinder(ViewGroup viewGroup, final CoordinatorProvider provider) {
+    for (int i = 0; i < viewGroup.getChildCount(); ++i) {
```
Comment (loganj) [inline coordinators/src/main/java/com/squareup/coordinators/Coordinators.java]: 1. please pull `viewGroup.getChildCount()` into a variable so it's only called once.
2. this is a nit, but generally we prefer `i++` to `++i`

**Comment by loganj** [comment]
That's me, sorry for the delay.  Can I ask you to sign our [Contributor License Agreement](https://spreadsheets.google.com/spreadsheet/viewform?formkey=dDViT2xzUHAwRkI3X3k5Z0lQM091OGc6MQ&ndplr=1), so that we can accept the change?

**Comment by loganj** [comment]
Actually, on closer look this is an incomplete fix. `installBinder` is meant to be recursive, and this will only bind immediate children.

**Comment by saantiaguilera** [comment]
Sorry, are you sure about that? It just adds a hierarchyListener to the `ViewGroup`, but as far as I remember, the listener only applies to the child added (not to the whole underlying view tree).

If you like I can also make install binder act on the whole tree (instead of just the `ViewGroup` childs)

**Comment by saantiaguilera** [comment]
@loganj Accepted the license and CR done!

**Comment by loganj** [comment]
You're right, I was misremembering. Never implemented the recursive behavior, decided to add it later if needed.

This looks great, thanks!

**Pull request merged**